build.xml

```diff
@@ -66,7 +66,7 @@
 
   <target depends="init" name="build">
     <echo message="${ant.project.name}: ${ant.file}"/>
-    <javac source="1.6" target="1.6" debug="true" debuglevel="${debuglevel}" destdir="${build.classes}" includeantruntime="false">
+    <javac source="7" target="7" debug="true" debuglevel="${debuglevel}" destdir="${build.classes}" includeantruntime="false">
```

Comment:
I think, it's fine to bump this to 8.

Reply:
Sure, I don't mind bumping to 8. I was going for the least invasive change as I only needed 7 for `ClassValue`, but if you want to establish a higher minimum I personally don't have anything against it.